core/src/test/java/org/apache/iceberg/actions/TestBinPackStrategy.java

```diff
@@ -33,6 +33,7 @@
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
+import org.assertj.core.util.Lists;
```

Comment:
Question / Nit: Is there a reason to use `Lists` from `assertj` instead of `Lists` from the relocated google commons library?

Reply:
oh nice catch, should not use that. We need to add it to checkstyle.